src/main/index.ts

```diff
@@ -41,8 +40,7 @@ async function main() {
   const updater = new AppUpdater()
   updater.start();
 
-  registerFileProtocol(appProto, app.getPath("userData"));
```

Comment:
Why `appProto` has been removed?

Reply:
Because it's never used

Reply:
Might be a dumb question but how do we display uploaded cluster icons then?

Reply:
I switched to saving the base64 encoding of them in the user store

Reply:
It removes the filename issue that we were seeing with some unicode characters, and it removes the security problems when running on dev mode.

Reply:
> I switched to saving the base64 encoding of them in the user store

Ok, do we have a migration for that?

Reply:
I did not code one up, I should do that then

Reply:
> I did not code one up, I should do that then

Yes we should have a migration. Otherwise Lens will drop all the existing cluster icons on upgrade.